dex/testing/zec/harness.sh

```diff
@@ -316,18 +316,20 @@ sleep 1
 
 tmux send-keys -t $SESSION:4 "./alpha z_importwallet ${SOURCE_DIR}/alphawallet ${DONE}" C-m\; ${WAIT}
 tmux send-keys -t $SESSION:4 "./beta z_importwallet ${SOURCE_DIR}/betawallet ${DONE}" C-m\; ${WAIT}
+tmux send-keys -t $SESSION:4 "./delta z_importwallet ${SOURCE_DIR}/deltawallet ${DONE}" C-m\; ${WAIT}
+tmux send-keys -t $SESSION:4 "./gamma z_importwallet ${SOURCE_DIR}/gammawallet ${DONE}" C-m\; ${WAIT}
 
 echo "Generating 400 blocks for alpha"
 tmux send-keys -t $SESSION:4 "./alpha generate 400${DONE}" C-m\; ${WAIT}
 
-#################################################################################
+################################################################################
 # Send gamma and delta some coin
 ################################################################################
 
 ALPHA_ADDR="tmEgW8c44RQQfft9FHXnqGp8XEcQQSRcUXD"
 BETA_ADDR="tmSog4freWuq1aC13yf1996fy4qXPmv3GTB"
-DELTA_ADDR=`./delta getnewaddress`
```

Comment:
We require stuff like perl in different places, so I don't think requiring `jq` is too much to ask if we want to eventually use the newer `z_` RPCs